.circleci/config.yml

```diff
@@ -104,40 +104,45 @@ jobs:
       - run:
           name: Linting
           command: |
-            [[ $CIRCLE_NODE_INDEX =~ [123] ]] || cargo x lint
-            [[ $CIRCLE_NODE_INDEX =~ [123] ]] || cargo xclippy --workspace --all-targets
-            [[ $CIRCLE_NODE_INDEX =~ [123] ]] || cargo fmt -- --check
+            [[ $CIRCLE_NODE_INDEX =~ [1234] ]] || cargo x lint
```

Comment:
@sunmilee It seems to me this would not run on the client-dev workspace, as x is not in client-dev workspace. Probably need to do the lint/fmt/clippy in client-dev separately in its own step below.

Reply:
I've removed the dedicated workspace, we don't need it until we start to publish real binaries.